draft-jholland-quic-multicast.md

```diff
@@ -191,6 +191,8 @@ The client tells its server about some restrictions on resources that it is capa
 The server asks the client to join channels with MC_JOIN ({{channel-join-frame}}) frames and to leave channels with MC_LEAVE ({{channel-leave-frame}}) frames.
 
 The server uses the MC_ANNOUNCE ({{channel-announce-frame}}) frame before any join or leave frames for the channel to describe the channel properties to the client, including values the client can use to ensure the server's requests remain within the limits it has sent to the server, as well as the secrets necessary to decode the headers of packets in the channel.
+This is done in advance of sending an MC_JOIN frame to make sure the client can establish necessary state for joining and retire any connection IDs that would be colliding with the channel ID.
+Separating announcement of channels from joining also allows clients to leave and later rejoin channels in cases where they get temporarily experience a high rate of loss or switch to a network that does not support multicast.
```

Comment:
```suggestion
Separating announcement of channels from joining them allows clients to leave and later rejoining channels at their discretion without the cost of terminating a connection. This could, for example, be used by clients to temporarily disabling multicast reception in response to a high rate of loss or a switch to a network that does not support multicast.
```